torch/nn/functional.py

```diff
@@ -406,6 +407,28 @@ def dropout(input, p=0.5, training=False, inplace=False):
     return _functions.dropout.Dropout(p, training, inplace)(input)
 
 
+def alpha_dropout(input, p=0.5, training=False):
+    if p < 0 or p > 1:
+        raise ValueError("dropout probability has to be between 0 and 1, "
+                         "but got {}".format(p))
+
+    if p == 0 or not training:
+        return input
+
+    alpha = -1.7580993408473766
```

Comment:
I've used the value from [original implementation by the authors](https://github.com/bioinf-jku/SNNs/blob/master/selu.py#L35), which seems to be the value so that the distributions are with mean 0 and std 1 after each layer. I could add arguments to enable the user to specify the desired mean/std, but is there an use-case for that?